### PR TITLE
fix foodq

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: generator-foodq/generators/app2/index.js [hunk 1]
```diff
@@ -3,13 +3,17 @@ var _ = require('lodash');
...
 const Datauri = require('datauri/sync');
 const DEFAULT_IMAGE = require("./images/defaultImage");
+const types = require('@sap-devx/yeoman-ui-types');
 
 module.exports = class extends Generator {
 	constructor(args, opts) {
 		super(args, opts);
 		this.prompts = opts.prompts;
+		this.appWizard = opts.appWizard;
 		this.parentPromptsQuantity = this.prompts.size();
```
Comment (tomer-epstein) [inline generator-foodq/generators/app2/index.js:13]: this.appWizard = types.AppWizard.create(opts);

File: generator-foodq/generators/app/index.js [hunk 2]
```diff
@@ -396,10 +408,13 @@ module.exports = class extends Generator {
...
 
 	install() {
+		this.log('FoodQ is installing dependencies.');
 		this.npmInstall(['lodash'], { 'save-dev': true });
 	}
 
 	end() {
+		this.log('FoodQ completed to install dependencies.');
+		this.log('FoodQ generation completed.');
 		this.appWizard.showInformation('FoodQ ended', types.MessageType.notification);
```
Comment (ira-gordin-sap) [inline generator-foodq/generators/app/index.js:418]: Paola asked to replace "FoodQ ended" with "The XXX project has been generated."

File: generator-foodq/generators/app2/index.js (hunk 1 above)
Comment (slavik-lvovsky) [reply generator-foodq/generators/app2/index.js:13]: This is incorrect, because you do not know the name of the object passed to a sub generator

**Review by ira-gordin-sap — CHANGES_REQUESTED** [review]

**Review by ira-gordin-sap — APPROVED** [review]

File: generator-foodq/generators/app2/index.js (hunk 1 above)
Comment (tomer-epstein) [reply generator-foodq/generators/app2/index.js:13]: **object name should always be appWizard!**

https://github.com/SAP/yeoman-ui/pull/391/files#diff-2aa9919e6c081b46db7b1e2ad5d72ecb045858204730797d687e6fac40d1773fL16

**Pull request merged**